nifi-nar-bundles/nifi-easyrules-bundle/nifi-easyrules-service/src/main/resources/docs/org.apache.nifi.rules.engine.EasyRulesEngineService/additionalDetails.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<head>
+    <meta charset="utf-8" />
+    <title>EasyRulesEngineService</title>
+    <link rel="stylesheet" href="../../../../../css/component-usage.css" type="text/css" />
+</head>
+<body>
+<h2>General</h2>
+<p>The Easy Rules Engine Service supports execution of a centralized set of rules (stored as files) against a provided set of data called facts.  Facts sent to the service are processed against
+    the rules engine to determine what, if any, actions should be executed based on the conditions defined within the rules. The list of actions are returned to the caller to handle as needed.
+</p>
+<p>
+    Rules files can be implemented in any of the following formats:
+</p>
+<ul>
+    <li> NiFi Rules Format - This is a rules file which follows the NiFi style for rules definition, which supports MVEL (MVFLEX) Expression language for conditions (default format).
+    </li>
+    <li> Easy Rules Format with MVEL Support - This is a rules file which follows the Easy Rules style for rules definition that supports MVEL (MVFLEX) Expression language for conditions and actions
+    </li>
+    <li> Easy Rules Format with SPEL Support - This is a rules file which follows the Easy Rules style for rules definition that supports Spring Expression language for conditions and actions
+    </li>
+</ul>
+<p>
+    All rules formats can be implemented as JSON or YAML files (with JSON serving as default file type).
+</p>
+
+<p>
+    NiFi Rules Format consist of the following elements:
+</p>
+<table>
+    <tr><td>Name</td><td>The name of the rule.</td></tr>
+    <tr><td>Description</td><td>A description of the rule.</td></tr>
+    <tr><td>Priority</td><td>Unique number for order in which rule should be executed in the rules engine.</td></tr>
+    <tr><td>Condition</td><td>The condition that will be checked given a set of fact data. For NiFi Format the MVEL expression language is supported.</td></tr>
+    <tr><td>Action(s)</td><td>The required actions that should be performed if the condition checked returns true.  This action includes the type of action and any attributes required to execute the action.</td></tr>
```

Comment:
Should we include some example actions here, such as EXPRESSION, ROUTE, SEND, LOG, etc.?

Reply:
Yep that makes sense.  I will add those in

Reply:
@mattyb149 I may lean here more on describing the way people can define actions which can be leveraged by their own handlers